src/components/__tests__/FontAwesomeLayers.test.js

```diff
@@ -2,10 +2,13 @@
  * @jest-environment jsdom
  */
 
-import { library } from '@fortawesome/fontawesome-svg-core'
-import { faCoffee, faCircle } from '../__fixtures__/icons'
 import { compileAndMount } from '../__fixtures__/helpers'
+import { faCoffee, faCircle } from '../__fixtures__/icons'
+import { library } from '@fortawesome/fontawesome-svg-core'
+import { ICON_PACKS_STARTING_VERSION, SVG_CORE_VERSION } from '../../utils'
```

Comment:
I added this import here and the "semver" import.
The other imports were previously added, I just alphabetized them.